src/main/resources/hudson/plugins/gradle/injection/InjectionConfig/config.jelly

```diff
@@ -0,0 +1,77 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:section title="${%Build Scans injection}">
+        <f:optionalBlock field="enabled" title="${%Enable Build Scans injection}" inline="true">
+
+            <f:section title="${%Gradle Enterprise Connection Settings}">
+                <f:entry title="${%Gradle Enterprise server URL}" field="server">
+                    <f:textbox checkMethod="post"/>
+                </f:entry>
+                <f:entry field="allowUntrusted">
+                    <f:checkbox title="${%Allow Untrusted Server}"/>
+                </f:entry>
+                <f:entry title="${%Gradle Enterprise Access Key}" field="accessKey">
+                    <f:textbox checkMethod="post"/>
```

Comment:
Please make this a password instead of textbox

Reply:
Done.